keystone-spassword.spec

```diff
@@ -63,12 +63,16 @@ pwd_max_tries=5
 pwd_block_minutes=30
```

Comment:
CNR entry missing 

(I know the PR is WIP and may pass some time until it gets ready to merge... but better comment this sooner than latter so I don't forget  about it ;)

Reply:
This PR is far to be ready for merge

Reply:
NTC (it was added in the last times)